internal/pkg/model/naming.go

```diff
@@ -143,22 +149,50 @@ func (n Naming) ConfigPath(parentPath string, component *Component, config *Conf
 		panic(fmt.Errorf(`config "%s" parent path cannot be empty"`, config))
 	}
 
+	// Shared code is handled differently
+	var template, targetComponentId string
+	if component.IsSharedCode() {
+		// Get target component ID for shared code config
+		if config.Content == nil {
+			panic(fmt.Errorf(`shared code config "%s" must have set key "%s"`, config.Desc(), ShareCodeTargetComponentKey))
+		}
+		targetComponentIdRaw, found := config.Content.Get(ShareCodeTargetComponentKey)
```

Comment:
V ceste ku shared code je `target_component_id`, tj. ID komponenty, pre ktoru je kod urceny.
To je ulozene v konfiguracii.
Kazda target komponenta (napr. `python transformacie`)  ma vlastnu konfiguraciu so vsetkymi shared codes (config rows).